lib/everypolitician.rb

```diff
@@ -151,6 +151,12 @@ def csv
     def [](key)
       raw_data[key]
     end
+
+    private
+
+    def parse_partial_date(date)
+      Date.new(*date.split('-').map(&:to_i))
```

Comment:
Oh, nice. I don't think I realised that Date.new could take a variable number of arguments like that.